netplan_cli/cli/state.py

```diff
@@ -59,6 +60,9 @@
     'vrf': 'vrf',
     'vxlan': 'tunnel',
 
+    # Used for wifi testing
+    'ieee80211_radiotap': 'wifi',
```

Comment:
question: Could you please provide some instruction of how this can be used for wifi testing? Either as part of the PR description or a code comment. I cannot see it being used in the automated tests.

Reply:
Yes, sorry. This is the device type for the device `hwsim0` created by the `mac80211_hwsim` module.
I used it to test my changes and noticed warnings from `netplan status`.

This is how `networkctl` sees it: `7 hwsim0 ieee80211_radiotap off         unmanaged`

I added it to the list to avoid this warning from `netplan status`:

```
~# netplan status
Unknown device type: ieee80211_radiotap
Unknown device type: ieee80211_radiotap
     Online state: online
...
```

I'll improve the comment.